evalml/tests/component_tests/decomposer_tests/test_polynomial_decomposer.py

```diff
@@ -286,7 +235,12 @@ def test_polynomial_decomposer_build_seasonal_signal(
     # and in the middle of a cycle
     y_test = y[test_first_index:]
 
-    projected_seasonality = decomposer._build_seasonal_signal(
+    # Set the decomposer's trend attribute since the function uses it to select the
+    # proper integer/datetime index.  The actual value doesn't matter, just that
+    # something with an index exists there.
+    decomposer.trend = full_seasonal_signal
```

Comment:
Correct me if I'm wrong, but isn't this the same test as was added to `test_decomposer`?

Reply:
You are not wrong!